Automattic/wp-calypso
# Add Wrapper around Cart Variant Picker

#### Changes proposed in this Pull Request

* Import organizational changes from #61174 to simplify that PR

#### Testing instructions

1. Boot branch or use live link as Calypso Blue.
2. Add a Jetpack product to cart.
3. Go to cart and verify that nothing has changed vs production.
4. Repeat steps 2. & 3. for a WordPress.com product.
